### PR TITLE
Get address summary

### PR DESCRIPTION
**Pull request by pnagurny**

### CODE REVIEW

File: lib/services/address/index.js [hunk 1]
```diff
@@ -871,4 +879,121 @@ AddressService.prototype.getAddressHistory = function(addresses, options, callba
...
 };
 
+/**
+ * This will return an object with:
+ *   balance - confirmed balance
+ *   unconfirmedBalance - unconfirmed balance
+ *   totalReceived - satoshis received
+ *   totalSpent - satoshis spent
+ *   appearances - number of times used in confirmed transactions
+ *   unconfirmedApearances - number of times used in unconfirmed transactions
```
Comment (braydonf) [inline lib/services/address/index.js]: typo

**Comment by kleetus** [comment]
LGTM, all tests pass locally.

**Comment by braydonf** [comment]
After further review, the total transaction count is incorrect.

**Comment by braydonf** [comment]
Spawn an issue regarding isSpent binding, as revealed here: https://github.com/bitpay/bitcore-node/issues/245

**Pull request merged**